lua/hawtkeys/init.lua

```diff
@@ -46,55 +46,49 @@ local _defaultSet = {
     }, -- method 6
 }
 
+M.defaultConfig = {
+    leader = " ",
+    homerow = 2,
+    powerFingers = { 2, 3, 6, 7 },
+    keyboardLayout = "qwerty",
+    keyMapSet = _defaultSet,
+    highlights = {
+        HawtkeysMatchGreat = { link = "DiagnosticOk", underline = true },
```

Comment:
This is not a valid highlight configuration. `link` is only functional on its own, and all other fields will be ignored as far as I know - we need to use `nvim_get_hl` to get the actual field values and then add underline (which is why we need the ColorScheme autocmd)

Reply:
Edit: I actually think this is cool. If we keep this, people can modify existing groups easily.